src/App.jsx

```diff
@@ -14,45 +16,104 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const FILTER_ALPHABETICALLY = 'Sort alphabetically';
+const FILTER_LENGTH = 'Sort by length';
+
+export const App = () => {
+  const [chosenFilter, setChosenFilter] = useState('');
+  const [reverseBtn, setReverseBtn] = useState(false);
```

Comment:
```suggestion
  const [isReversed, setIsReversed] = useState(false);
```
for boolean names: https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig